### PR TITLE
Update docs and template for CF deployment

### PR DESCRIPTION
**Pull request by petergtz**
We've used this updated cf template manifest and the updated README to do a successful deployment in Softlayer.

/cc @suhlig @smoser-ibm

### CODE REVIEW

**Comment by maximilien** [comment]
Thanks for contribution @petergtz 

/cc @gu-bin I am asking you to review and if all OK let's merge.

File: docs/mimimal-softlayer.yml [hunk 1]
```diff
@@ -366,6 +307,62 @@ jobs:
   properties:
     login:
       catalina_opts: -Xmx768m -XX:MaxPermSize=256m
+      saml:
+        serviceProviderCertificate: |
+          -----BEGIN CERTIFICATE-----
+          MIIENjCCAh6gAwIBAgIQKKBo+285koaiAaJKVNstEzANBgkqhkiG9w0BAQsFADAZ
```
Comment (maximilien) [inline docs/mimimal-softlayer.yml]: Should we remove certificates in sample manifests?

File: docs/mimimal-softlayer.yml [hunk 2]
```diff
@@ -366,6 +307,62 @@ jobs:
...
+          5o6v40kRkC42cbda2WFDc5YGgltIUi7fpEErHg+XnTzoJULsYm1wlorsee4x5pnz
+          SOr9lV8awgwrRxaoZHzQ4LmcQthW5ZVgrKF707OSmGL3317t4SRUIrBirWYb9oco
+          VSdZ4SDPK2lUFb5ZLXlOlq3rak8KVIUxSpbiBGbuz0V7hqJscYWPD0dWNYE7B9eR
+          aXLRafZQ9prE8CuW9bnShty0zyht/fLlRYY=
+          -----END CERTIFICATE-----
+        serviceProviderKey: |
+          -----BEGIN RSA PRIVATE KEY-----
+          MIIEpAIBAAKCAQEAvdIGFR27voGePkj5wPY/OwcIkuIjaNeMxLAXiki4NvzpjdXv
+          +FNhH2N1GwDP/qtnDvF8z97SMhTw83z7iLTu0WrVq/ccXg9nL76tz74pGpQ9732Q
+          D2Ou2FOwMs1qOlUhxZMaILfTjOBVzzp3FT5WXaT5BJn4ENfOTwZU8PW4IBbclY3g
```
Comment (maximilien) [inline docs/mimimal-softlayer.yml]: Same for sample keys or is it clear that this needs to change?

File: docs/mimimal-softlayer.yml [hunk 3]
```diff
@@ -1,117 +1,74 @@
...
 releases:
 - {name: cf, version: latest}
-- {name: postgres, version: latest}
 - {name: diego, version: latest}
-- {name: etcd, version: latest}
-- {name: garden-linux, version: latest}
+- {name: garden-runc, version: latest}
 - {name: cflinuxfs2-rootfs , version: latest}
 
 networks:
```
Comment (gu-bin) [inline docs/mimimal-softlayer.yml]: I think we need to put an example of manual network here.

File: docs/mimimal-softlayer.yml (hunk 1 above)
Comment (gu-bin) [inline docs/mimimal-softlayer.yml]: Yes, I think so

Comment (petergtz) [inline docs/mimimal-softlayer.yml]: Note that https://github.com/cloudfoundry/cf-release/blob/master/example_manifests/minimal-aws.yml contains sample manifests too. We tried to stick with that one as close as possible.

File: docs/mimimal-softlayer.yml (hunk 3 above)
Comment (petergtz) [inline docs/mimimal-softlayer.yml]: Interestingly, no, we don't. For some reason, even though cf_private is referenced in 3 other locations, it's not really used. This manifest is tested and works as is. But you're right, it's confusing. I'll try to remove them and test again.

File: docs/mimimal-softlayer.yml (hunk 2 above)
Comment (petergtz) [inline docs/mimimal-softlayer.yml]: See above.

File: docs/mimimal-softlayer.yml (hunk 1 above)
Comment (maximilien) [inline docs/mimimal-softlayer.yml]: OK, @petergtz but clearly these certificate should not be certificates used in any parts of your testing or other envs. Goes w/o saying and just confirming. Thx

**Comment by maximilien** [comment]
OK I am happy to merge once @gu-bin approves the changes addressing her comments.

**Comment by maximilien** [comment]
@gu-bin can you please complete your review.

@petergtz seems like there is a conflict. Perhaps a docs file changed. Please merge and resubmit. Thanks.

**Comment by gu-bin** [comment]
@petergtz Please remove `cf_private` network references as I commented before.

**Comment by petergtz** [comment]
@gu-bin yes, will do. Sorry, we had/have some high priority items on our backlog which is why we couldn't address your comment yet. Will do so as soon as possible though. Thanks.

**Comment by petergtz** [comment]
@gu-bin @maximilien Removed references to `cf_private` and resolved merging conflicts, which were both blocking the merge.

/cc @suhlig @smoser-ibm

**Pull request merged**